src/oc/nrnran123.cpp

```diff
@@ -92,7 +92,7 @@ extern "C" double nrnran123_negexp(nrnran123_State* s) {
 }
 
 /* at cost of a cached  value we could compute two at a time. */
-double nrnran123_normal(nrnran123_State* s) {
+extern "C" double nrnran123_normal(nrnran123_State* s) {
```

Comment:
Is this really necessary.  I would think that the extern "C" in the nrnran123.h file would be sufficient.

Reply:
Although, I do see all the others have the same style. Is that the recommended style?

Reply:
IIRC, I had some issues on a platform. Apart from that, it does make it easier to know we have an exported function .